packages/asc-ui/src/Radio/Radio.stories.tsx

```diff
@@ -1,11 +1,12 @@
 import * as React from 'react'
 import { storiesOf } from '@storybook/react'
-import { Radio } from '@amsterdam/asc-core'
+import { ButtonBar } from '@amsterdam/asc-core'
+import Radio from './Radio'
 
-storiesOf('Radio', module).add('All Radios', () => (
+storiesOf('Radio', module).add('All radios', () => (
   <div>
-    <Radio id="radio-1" name="gender" value="female" label="Vrouw" defaultChecked />
-    <Radio id="radio-2" name="gender" value="male" label="Man" />
-    <Radio id="radio-3" name="gender" value="none" label="Onzijdig" disabled />
+    <Radio id="female" name="gender" value="female" label="Vrouw" />
```

Comment:
I think we can leave out the id, right? Since id is always the same as value, you can pass value to prop id in ./Radio

Reply:
sorry, I wasnt clear what I meant hehe, I think in this case the user (read: developer) shouldn't worry about any id, since we can handle the connection between for & id in the component. We just have to make sure we generate a unique id, in case someone uses the same value on the same page.